src/routes/UserRoute.ts

```diff
@@ -1 +1,8 @@
-export {};
+import { Router } from 'express'
+
+const userRouter = Router()
+
+userRouter.get('user')
+userRouter.post('user')
```

Comment:
what is the point of these! your task requires only one endpoint to login users!